tfe/resource_tfe_workspace.go

```diff
@@ -85,7 +86,7 @@ func resourceTFEWorkspace() *schema.Resource {
 			"execution_mode": {
 				Type:          schema.TypeString,
 				Optional:      true,
-				Computed:      true,
```

Comment:
I wonder why this was listed as `Computed`? Any thoughts?

Reply:
This is most likely based on logic that whenever an execution_mode is not specified, it will be computed and returned. With this changes, the default value is always appended whenever the field is not specified.

Reply:
Actually, Computed might still needed for the `execution_mode` to be updated when it changes as a result of `operation` field change. I am investigating this.

Reply:
Looks like `operations` attribute has been marked as deprecated. Would it be easier to reason about if we removed the attribute?

Reply:
With the new changes, I have marked it as deprecated to match the API docs.